src/main/java/core/basesyntax/OperationStrategy.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public class OperationStrategy {
+    public void execute(ShopStorage storage, FruitTransaction transaction) {
+    }
+}
```

Comment:
The `OperationStrategy` class should be an interface or an abstract class, and it should not directly implement the `execute` method. Instead, it should define a method to get the appropriate `OperationHandler` based on the `FruitTransaction.Operation`. Consider refactoring this class to follow the strategy pattern properly.